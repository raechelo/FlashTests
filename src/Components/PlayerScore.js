import React, { Component } from 'react';


export default class PlayerScore extends Component {
  constructor() {
    super();
    this.state = {

    }
  }
  render() {
    return <div className="scoreboard">
            <h2 className="player">Player Score</h2>
            <h3 className="p-score">0</h3>
          </div>
  }
}