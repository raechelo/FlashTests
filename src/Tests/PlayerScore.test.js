import React from 'react';
import PlayerScore from '../Components/PlayerScore';
import { shallow } from 'enzyme';

describe('PlayerScore', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <PlayerScore />
    )
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
  
})