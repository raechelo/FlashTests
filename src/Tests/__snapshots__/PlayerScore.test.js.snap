// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerScore should match the snapshot with all the data passed in 1`] = `
<div
  className="scoreboard"
>
  <h2
    className="player"
  >
    Player Score
  </h2>
  <h3
    className="p-score"
  >
    0
  </h3>
</div>
`;
